src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -508,6 +522,34 @@ bool WholeBodyDynamicsDevice::openExternalWrenchesPorts(os::Searchable& config)
     return ok;
 }
 
+bool WholeBodyDynamicsDevice::openFilteredFTPorts(os::Searchable& config)
+{
+    bool ok = true;
+        // create port names
```

Comment:
Please indent consistently the code with four spaces

Reply:
Done